src/main/java/codeu/controller/ProfilePageServlet.java

```diff
@@ -1,20 +1,66 @@
 package codeu.controller;
 
+import codeu.model.store.basic.UserStore;
+import codeu.model.store.basic.UserProfileStore;
 import java.io.IOException;
 import javax.servlet.ServletException;
 import javax.servlet.http.HttpServlet;
 import javax.servlet.http.HttpServletRequest;
 import javax.servlet.http.HttpServletResponse;
 
 /**
- * Servlet class responsible for User profile pages.
+ * Servlet class responsible for User Profile pages.
  */
 public class ProfilePageServlet extends HttpServlet {
 
+  /** Store class that gives access to Users and User Profiles. */
+  private UserStore userStore;
+  private UserProfileStore userProfileStore;
+
+  /**
+   * Set up state for handling profilepage-related requests. This method is only called when running in a
+   * server, not when running in a test.
+   */
+  @Override
+  public void init() throws ServletException {
+    super.init();
+    setUserStore(UserStore.getInstance());
+    setUserProfileStore(UserProfileStore.getInstance());
+  }
+
+  /**
+   * Sets the UserStore used by this servlet. This function provides a common setup method for use
+   * by the test framework or the servlet's init() function.
+   */
+  void setUserStore(UserStore userStore) {
+    this.userStore = userStore;
+  }
+
+  /**
+   * Sets the UserProfileStore used by this servlet. This function provides a common setup method for use
+   * by the test framework or the servlet's init() function.
+   */
+  void setUserProfileStore(UserProfileStore userProfileStore) {
+    this.userProfileStore = userProfileStore;
+  }
+
+  /**
+   * This function fires when a user requests the /profilepage URL. It simply forwards the request to
+   * profilepage.jsp.
+   */
   @Override
   public void doGet(HttpServletRequest request, HttpServletResponse response) 
     throws IOException, ServletException { 
     request.getRequestDispatcher("/WEB-INF/view/profilepage.jsp").forward(request, response);
   }
 
+  /**
+   * This function fires when a user posts the /profilepage URL. It simply forwards the request to
```

Comment:
Update comment (this method is going to do more than just delegate to profilepage.jsp). Since this doesn't have a fleshed-out implementation yet, you could also add a `TODO` comment sketching out what this method should do.

It could also make sense to separate the changes to this file from the datastore implementation, and not touch this until after those are submitted and we're ready to implement the body of the doPost method. That way the changes adding the data store variables and the code using and testing those datastores is all in one PR.